common/src/db/query.rs

```diff
@@ -473,6 +474,7 @@ impl Arg {
             return Ok(Arg::Null);
         }
         Ok(match ct {
+            ColumnType::Uuid => Arg::Value(Value::from(s.parse::<Uuid>().map_err(err)?)),
```

Comment:
Ideally, we'd have a test for this in this file. Can you change the type for `id` from `i32` to `Uuid` in the dummy advisory model at the bottom of this file, and re-run the tests? I expect you'll need to add some quotes around some of the expected where clauses.

If that's too big of a hassle, I'm happy to do it after this PR merges.

Reply:
will add!